charts/collector/values.yaml

```diff
@@ -31,6 +31,10 @@ collector:
   service:
     type: ClusterIP
     port: 5000
+  # Set this to the number of requests that will remain in the PostGresDB post daily cleaning
+  max_har_storage: 2000000
+  # Set this to the number of consequetive minutes of abnormal traffic needed to triggerr a traffic alert
+  traffic_alert_min: 1
```

Comment:
typos: consecutive, trigger, PostgreSQL

I feel as if these descriptions could use an iteration or two, with someone outside of the team, to avoid the reader needing to understand jargon about a system they haven't seen, eg. abnormal traffic

Reply:
@lena-larionova ? ^^

Reply:
>  # Set this to the number of requests that will remain in the PostGresDB post daily cleaning

Suggestion: Set this to the maximum number of requests to keep in the PostgreSQL database. Any requests over this number are cleaned out daily, starting from the oldest request.

>   # Set this to the number of consequetive minutes of abnormal traffic needed to triggerr a traffic alert
Suggestion: Set this to the number of allowed consecutive minutes of irregular traffic. If traffic remains abnormal past the set limit, it triggers an alert. 

Agree with @wongstein - what exactly is abnormal/irregular traffic? I don't really know how to help explain it without some background here. Is there another setting somewhere that defines "abnormal"?

Reply:
@lena-larionova @jackkav I guess abnormal/irregular traffic is the traffic alert.  So this variable (traffic_alert_min) was created to prevent some alert fatigue.  We look at traffic every minute, and if in that minute the traffic looks weird we CAN send a traffic alert.  BUT traffic can be strangely irregular and some people don't care about just one minute of weird traffic.  They want to see at least 5 minutes of weird traffic continuously before they think it's worth of a traffic alert.  So that's what this variable is trying to cover.  To let the user say "Nah I don't want to be alerted unless there's been 3 minutes of weird traffic".